zilencer/management/commands/render_messages.py

```diff
@@ -47,9 +47,14 @@ def handle(self, *args, **options):
             result.write('[')
             messages = Message.objects.filter(id__gt=latest - amount, id__lte=latest).order_by('id')
             for message in queryset_iterator(messages):
+                content = message.content
+                if message.edit_history:
+                    history = ujson.loads(message.edit_history)
+                    history = sorted(history, key=lambda i: i['timestamp'], reverse=True)
+                    content = history[-1]['prev_content']
```

Comment:
as I found these items in edit_history already sorted that way, so the first entry is in the end of the list. Thus, this sorting procedure may be redundant, but I leave it just to be sure we get earliest message.